pkg/webhook/validation/uservalidation.go

```diff
@@ -44,7 +44,8 @@ const (
 )
 
 var (
-	fleetCRDGroups = []string{"networking.fleet.azure.com", "fleet.azure.com", "multicluster.x-k8s.io", "cluster.kubernetes-fleet.io", "placement.kubernetes-fleet.io"}
+	fleetDataplaneLabelPrefix = "cluster.kubernetes-fleet.io/"
```

Comment:
```suggestion
	fleetDataplaneLabelPrefix = "kubernetes-fleet.io/"
```

Reply:
cluster.kubernetes-fleet.io was already agreed and is already implemented on the RP side.